IdokladSdk/Models/IssuedInvoice/Get/IssuedInvoiceItemGetModel.cs

```diff
@@ -5,7 +5,7 @@ namespace IdokladSdk.Models.IssuedInvoice
     /// <summary>
     /// Issued invoice item.
     /// </summary>
-    public class IssuedInvoiceItemGetModel : IssuedInvoiceItemListGetModel
+    public class IssuedInvoiceItemGetModel : IssuedInvoiceItemListGetModel, IEntityId
```

Comment:
Ten interface vyhodit, implementuje ho předek (IssuedInvoiceItemListGetModel), takže zde je to zbytečné.